### DISCUSSION TITLE
Multiple Themes management in ThemeProvider

### DISCUSSION THREAD

**Discussion by oklex** [heroui-inc/heroui#815] [Feature requests]
I would love the ability to handle more color themes than just light and dark in the theme provider.

**Comment by tianenpang** [comment]
Hi @oklex you can use [`next-themes`](https://github.com/pacocoursey/next-themes) with [`NextUI.createTheme`](https://nextui.org/docs/theme/customize-theme) to implement it easily 🚀 

Here's an example: [codesandbox.io/s/nextui-discussion-815-multi-themes-yyb5ee](https://codesandbox.io/s/nextui-discussion-815-multi-themes-yyb5ee?file=%2Ftheme.js)
